tests/Model/ModelManagerTest.php

```diff
@@ -724,6 +724,29 @@ public function testGetEntityManagerException(): void
         $this->modelManager->getEntityManager(VersionedEntity::class);
     }
 
+    public function testGetNewFieldDescriptionInstance(): void
```

Comment:
Could we add a test for the case where the name is set explicitly to a value different than its default?

Reply:
Done @phansys